File: app/src/main/java/com/jikexueyuan/mobile/address/StringResponseBodyConverter.java
package com.jikexueyuan.mobile.address;

import com.squareup.okhttp.ResponseBody;

import java.io.IOException;

import retrofit.Converter;

/**
 * Created by aven on 10/22/15.
 */
public class StringResponseBodyConverter implements Converter<ResponseBody, String> {

    @Override
    public String convert(ResponseBody value) throws IOException {
        return new String(value.bytes(), "UTF-8");
    }
}

File: app/src/main/kotlin/com/jikexueyuan/mobile/address/api/StringResponseBodyConverter.kt
package com.jikexueyuan.mobile.address.api

import com.squareup.okhttp.ResponseBody

import java.io.IOException

import retrofit.Converter

/**
 * Created by aven on 10/22/15.
 */
class StringResponseBodyConverter : Converter<ResponseBody, String> {

    @Throws(IOException::class)
    override fun convert(value: ResponseBody): String {
        return String(value.bytes(), "UTF-8")
    }
}
